documentation/source/getting_started/quick_install.rst

```diff
@@ -0,0 +1,23 @@
+You can install OpenEBS instantly using the following procedure.
+
+Prerequisite:
+-------------
+You must have Kubernetes installed.
```

Comment:
(Let us make prerequisite as two steps )
1. You must have kubernetes installed
2. You must have iscsi initiator enabled on your minion nodes. The instructions to install / setup iscsi initiator can change depending on your host OS
(Follow 2nd point with example of installing open-iscsi on ubuntu)